src/main/kotlin/rx/lang/kotlin/observables.kt

```diff
@@ -107,3 +87,12 @@ fun <T, R> List<Observable<T>>.zip(zipFunction: (args: List<T>) -> R): Observabl
  * Returns an Observable that emits the items emitted by the source Observable, converted to the specified type.
  */
 inline fun <reified R : Any> Observable<*>.cast(): Observable<R> = cast(R::class.java)
```

Comment:
I'm kind of torn on this again. Keep going back and forth. Okay let's keep it.  Don't see any harm.